addons/account_peppol/tests/test_peppol_participant.py

```diff
@@ -127,6 +125,7 @@ def test_create_success_sender(self):
         # after running the cron checking participant status
         company = self.env.company
         wizard = self.env['peppol.registration'].create(self._get_participant_vals())
+        wizard.smp_registration = False  # by default, we try to register as receiver
```

Comment:
You can `create` directly the wizard with the right value for `smp_registration`